atom/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -379,6 +379,13 @@ OffScreenRenderWidgetHostView::~OffScreenRenderWidgetHostView() {
 #if defined(OS_MACOSX)
   DestroyPlatformWidget();
 #endif
+
+  if (copy_frame_generator_.get())
+    copy_frame_generator_.reset(nullptr);
+
+  delegated_frame_host_.reset(nullptr);
+  compositor_.reset(nullptr);
+  root_layer_.reset(nullptr);
```

Comment:
These classes are managed by `unique_ptr`s, so they are automatically deleted and we don't need to manually reset them here.